packages/core/src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -159,7 +159,18 @@ interface Environment {
   onAllowKubeResource: () => void;
 }
 
-export const getApplicationBuilder = () => {
+export interface ApplicationBuilderOptions {
+  /**
+   * @default true
+   */
+  fakeTime?: boolean | FakeTimersConfig;
+}
+
+export const getApplicationBuilder = (options: ApplicationBuilderOptions = {}) => {
+  const {
+    fakeTime = true,
+  } = options;
+
```

Comment:
I don't think that there's case where we want to support behavioural unit tests without fake time.